solar_crypto/transactions/builder/transfer.py

```diff
@@ -7,28 +7,33 @@ class Transfer(BaseTransactionBuilder):
 
     transaction_type = TRANSACTION_TRANSFER
 
-    def __init__(self, recipientId, amount, vendorField=None, fee=None):
+    def __init__(self, vendorField=None, fee=None):
         """Create a transfer transaction
 
         Args:
-            recipientId (str): address to which you want to send coins
-            amount (int): amount of coins you want to transfer
             vendorField (str): value for the vendor field aka smartbridge
             fee (int, optional): fee used for the transaction (default is already set)
         """
         super().__init__()
 
-        if not address.validate_address(recipientId):
-            raise ValueError("Invalid recipient address")
-
-        self.transaction.recipientId = recipientId
+        self.transaction.typeGroup = self.get_type_group()
 
-        if type(amount) == int and amount > 0:
-            self.transaction.amount = amount
-        else:
-            raise ValueError("Amount is not valid")
+        self.transaction.asset["transfers"] = []
 
         self.transaction.vendorField = vendorField.encode() if vendorField else None
 
         if fee:
             self.transaction.fee = fee
+
+    def get_type_group(self):
+        return TRANSACTION_TYPE_GROUP.CORE.value
+
+    def add_payment(self, amount, recipient_id):
```

Comment:
Is there any sort of aliasing/backwards compatibility where this (`def add_payment(self, amount, recipient_id):`) shouldn't just be removed altogether?

Reply:
Only reason why I added it was to have the same methods as core. I'll remove it.

Reply:
Right on. TS allows 'multipayment' to be an alias for 'transfer' (i.e., backwards compatibility). If that's not a thing here, it's not needed 👍

Thank you, @c0nsol3 🙏